WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationWizardContent.swift

```diff
@@ -143,25 +216,111 @@ extension SiteInformationWizardContent: UITableViewDataSource {
         return cell
     }
 
-    func tableView(_ tableView: UITableView, titleForFooterInSection section: Int) -> String? {
-        return TableStrings.footer
-    }
-
     private func configure(_ cell: InlineEditableNameValueCell, index: IndexPath) {
         if Rows.title.matches(index.row) {
             cell.nameLabel.text = TableStrings.site
-            cell.valueTextField.placeholder = TableStrings.site
+            cell.valueTextField.attributedPlaceholder = attributedPlaceholder(text: TableStrings.site)
+            cell.addTopBorder(withColor: WPStyleGuide.greyLighten20())
         }
 
         if Rows.tagline.matches(index.row) {
             cell.nameLabel.text = TableStrings.tagline
-            cell.valueTextField.placeholder = TableStrings.taglinePlaceholder
+            cell.valueTextField.attributedPlaceholder = attributedPlaceholder(text: TableStrings.taglinePlaceholder)
+            cell.addBottomBorder(withColor: WPStyleGuide.greyLighten20())
         }
+
+        cell.nameLabel.font = WPStyleGuide.fontForTextStyle(.body, fontWeight: .regular)
+        cell.nameLabel.textColor = WPStyleGuide.darkGrey()
+
+        cell.valueTextField.font = WPStyleGuide.fontForTextStyle(.body, fontWeight: .regular)
+        cell.valueTextField.textColor = WPStyleGuide.greyDarken30()
+
+        if cell.delegate == nil {
+            cell.delegate = self
+        }
+    }
+
+    private func attributedPlaceholder(text: String) -> NSAttributedString {
+        let attributes: [NSAttributedString.Key: Any] = [
+            .foregroundColor: WPStyleGuide.grey(),
+            .font: WPStyleGuide.fontForTextStyle(.body, fontWeight: .regular)
+        ]
+
+        return NSAttributedString(string: text, attributes: attributes)
+    }
+}
+
+extension SiteInformationWizardContent: InlineEditableNameValueCellDelegate {
+    func inlineEditableNameValueCell(_ cell: InlineEditableNameValueCell,
+                                      valueTextFieldDidChange valueTextField: UITextField) {
+        updateButton()
+    }
+
+    private func updateButton() {
+        formIsFilled() ? setupButtonAsNext() : setupButtonAsSkip()
+    }
+
+    private func formIsFilled() -> Bool {
+        return !titleString().isEmpty || !taglineString().isEmpty
     }
 }
 
-extension SiteInformationWizardContent: UITableViewDelegate {
-    func tableView(_ tableView: UITableView, willDisplayFooterView view: UIView, forSection section: Int) {
-        WPStyleGuide.configureTableViewSectionFooter(view)
+extension SiteInformationWizardContent {
+    private func startListeningToKeyboardNotifications() {
```

Comment:
I've seen a class called `KeyboardDismissHelper` used to help with some of this functionality? Did you have a chance to evaluate it for reuse in this case?

Reply:
I did, and I decided to not use it in this case for three reasons. First, this comment: `Meant initially to be used along with ReplyTextView`,  second because it seemed to be like it was a bit too much, and third because it didn't seem to be used too much (which is probably the weakest of the three arguments)

Reply:
Fair enough. `NUXKeyboardResponder` also seems to have some overlap, but it resides in `WordPressAuthenticator` at the moment.